projects/plugins/jetpack/extensions/blocks/blogroll/icon.js

```diff
@@ -1,7 +1,13 @@
 import { SVG, Path } from '@wordpress/components';
 
 export default (
-	<SVG height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" fill="#000000">
-		<Path d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm6 9.8a.9.9 0 0 1-.1.5l-2.1 4.9a1.34 1.34 0 0 1-1.3.8H9a2 2 0 0 1-2-2v-5a1.28 1.28 0 0 1 .4-1L12 5l.69.69a1.08 1.08 0 0 1 .3.7v.2L12.41 10H17a1 1 0 0 1 1 1z" />
+	<SVG width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<Path d="M4.08691 16.2412H20.0869V14.7412H4.08691V16.2412Z" />
+		<Path d="M4.08691 20.2412H13.0869V18.7412H4.08691V20.2412Z" />
+		<Path
+			fill-rule="evenodd"
+			clip-rule="evenodd"
```

Comment:
Shouldn't these be `fillRule` and `clipRule`?

Reply:
I believe they should be. 592958bafd254fcb23e69b956cc9e8c4e58845f0